src/ports/nova/templates/AuraTheme.novaextension/Themes/Aura Theme.css

```diff
@@ -0,0 +1,413 @@
+meta {
+    -theme-interface-style: dark;
+    -theme-vibrancy: none;
+    -theme-accent-color: false;
+}
+
+
+/*
+ * Window styles
+ */
+meta.window {
+    background-color: #110F18;
```

Comment:
In general, colors in the template folder should be tokens (color accents), we should not put the hex values manually.

Take a look at [aura tokenize](https://github.com/daltonmenezes/aura-theme/blob/main/docs/CLI.md) command line helper to convert the hex values used by Aura to the right color tokens.